Lib/pydoc.py

```diff
@@ -2503,7 +2505,7 @@ def html_topicpage(topic):
             xrefs = sorted(xrefs.split())
 
             def bltinlink(name):
-                return '<a href="topic?key=%s">%s</a>' % (name, name)
+                return '<a href="topic?key=%s">%s</a>' % (html_escape(name), html_escape(name))
```

Comment:
[PEP 8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) states that lines should be limited to 79 chars.